propagation/toolboxcloud.md

```diff
@@ -1,10 +1,25 @@
-### Toolbox Cloudification Service
+### Toolbox Cloudification Service 
 
-The Toolbox Cloudification Service focuses on porting functionality from existing (open source) toolboxes into the EO platform
-targeted by your project. This service is designed to enhance the scalability and integration of your toolbox's features
-within an APEx compatible cloud environment.
+The toolbox cloudification service makes available single data processing (not visualization) functions of existing toolboxes on APEx. These data processing operations can then be freely combined with other algorithms hosted on APEx. Instead of a file-centric and local processing scheme of a traditional toolbox, key operations can be performed in the cloud where EO data archives are directly available as data cubes, where arbitrary extends in time and space can be extracted. Furthermore, functions from various toolboxes can be combined in a single workflow through APEx. 
```

Comment:
Suggestion: The toolbox cloudification service extracts single data processing functions from existing toolboxes as makes them available as reusable, cloud-based APEx-compliant algorithms. It is important to note that this cloudification service focuses solely on the data processing capabilities of toolboxes and does not include visualization features.

Reply:
> Instead of a file-centric and local processing scheme of a traditional toolbox, key operations can be performed in the cloud where EO data archives are directly available as data cubes, where arbitrary extends in time and space can be extracted.

@p3dr0  - Would this also be the case for AP?

Reply:
> Furthermore, functions from various toolboxes can be combined in a single workflow through APEx. 

Suggestion: This approach ensures that capabilities from various toolboxes can be seamlessly combined with other APEx-compliant algorithms.